task3-4/task3-4/MainScreen/MainScreenViewController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ViewController.swift
+//  task3-4
+//
+//  Created by Григоренко Александр Игоревич on 23.09.2022.
+//
+
+import UIKit
+
+/// VC экрана с заказом
+class MainScreenViewController: UIViewController {
+    private var goToCheckScreen: UIAlertController = UIAlertController()
+    override func viewDidLoad() {
```

Comment:
```suggestion 

    override func viewDidLoad() {
```
отделяем свойства от жизненного цикла и методы друг от друга